src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -525,14 +524,13 @@ describe('Schemaform <SaveInProgressIntro>', () => {
         user={user}
         fetchInProgressForm={fetchInProgressForm}
         removeInProgressForm={removeInProgressForm}
-        renderSignInMessage={renderSpy}
         toggleLoginModal={toggleLoginModal}
         formConfig={formConfig}
-        headingLevel="h1"
+        headingLevel={1}
       />,
     );
 
-    expect(tree.find('h1')).to.exist;
```

Comment:
this assertion always passes. you can say `expect(tree.find('blahblahblah')).to.exist;` and it'll pass

Reply:
Good catch! This isn't obvious (to me, at least :upside_down_face: )